decentraland/unity-renderer
# Fix: fixed user context menu positioning in chat

## What does this PR change?

Fix #2882 
Forces the user context menu to stay inside screen

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/profile-ui-bounds
2. Go to a private notification
3. Right click on a received message
4. Verify the context menu is inside the screen

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
